frontend/src/pages/data/collaborators/collaborators.js

```diff
@@ -81,8 +84,11 @@ export default {
                 this.errors.username = "Username required";
             if (!this.currentCollab.password)
                 this.errors.password = "Password required";
-
-            if (this.errors.lastname || this.errors.firstname || this.errors.username || this.errors.password)
+            if (!this.currentCollab.email)
+                this.errors.email = "Email required";
+            if (!this.currentCollab.phone)
+                this.errors.phone = "Phone required";
```

Comment:
Are you sure that email and phone number should be mandatory? It is a nice addition to the template, but you should, in my opinion, be able to use pwndoc without a phone and an email since it is not used elsewhere